util.py

```diff
@@ -1,4 +1,7 @@
+import pathlib
+from pathlib import Path, PurePath, PurePosixPath, PureWindowsPath
 import os
+from os import path

```

Comment:
It is not a good idea to have both an object named "path" and one named "Path" in this namespace. Remove this line and change references to "path" to instead use "os.path"